tests/grpc/config/schema-config.json

```diff
@@ -0,0 +1 @@
+{"name":"config","version":"0.1.0","config":{"variables":{"a":{"type":"array","items":{"type":"string"}},"an":{"type":"array","items":{"$ref":"#/types/config:index:ConfigNested"}},"b":{"type":"boolean","defaultInfo":{"environment":["BOOL"]}},"db":{"type":"boolean","default":true},"di":{"type":"integer","default":42},"ds":{"type":"string","default":"defString"},"i":{"type":"integer","defaultInfo":{"environment":["INT"]}},"m":{"type":"object","additionalProperties":{"type":"string"}},"n":{"$ref":"#/types/config:index:ConfigNested"},"s":{"type":"string","defaultInfo":{"environment":["STRING"]}}},"defaults":["s","b","i","m","a","n","an"]},"types":{"config:index:ConfigNested":{"properties":{"a":{"type":"array","items":{"type":"string"}},"b":{"type":"boolean"},"i":{"type":"integer"},"m":{"type":"object","additionalProperties":{"type":"string"}},"s":{"type":"string"}},"type":"object","required":["s","b","i","m","a"]}},"provider":{"properties":{"a":{"type":"array","items":{"type":"string"}},"an":{"type":"array","items":{"$ref":"#/types/config:index:ConfigNested"}},"b":{"type":"boolean","defaultInfo":{"environment":["BOOL"]}},"db":{"type":"boolean","default":true},"di":{"type":"integer","default":42},"ds":{"type":"string","default":"defString"},"i":{"type":"integer","defaultInfo":{"environment":["INT"]}},"m":{"type":"object","additionalProperties":{"type":"string"}},"n":{"$ref":"#/types/config:index:ConfigNested"},"s":{"type":"string","defaultInfo":{"environment":["STRING"]}}},"required":["s","b","i","m","a","n","an"],"inputProperties":{"a":{"type":"array","items":{"type":"string"}},"an":{"type":"array","items":{"$ref":"#/types/config:index:ConfigNested"}},"b":{"type":"boolean","defaultInfo":{"environment":["BOOL"]}},"db":{"type":"boolean","default":true},"di":{"type":"integer","default":42},"ds":{"type":"string","default":"defString"},"i":{"type":"integer","defaultInfo":{"environment":["INT"]}},"m":{"type":"object","additionalProperties":{"type":"string"}},"n":{"$ref":"#/types/config:index:ConfigNested"},"s":{"type":"string","defaultInfo":{"environment":["STRING"]}}},"requiredInputs":["s","b","i","m","a","n","an"]},"resources":{"config:index:Get":{"properties":{"config":{"type":"string"}},"required":["config"]}}}
```

Comment:
https://go-review.googlesource.com/c/go/+/23353/3/src/cmd/go/test.go

There're some special treatments given to "testdata" folder, and while not necessary here, perhaps  can consider adopting to be conventional?

Reply:
This isn't testdata, it's just the provider's schema. We seem to have a convention of checking that in (which is good IMO) so I did that here, too.

Reply:
Ah.. it's hard to read on one line, consider indenting or using YAML. Just a nit though of course.